cookbooks/cdo-apps/recipes/default.rb

```diff
@@ -109,7 +110,6 @@
     # Or install nodejs if the daemon builds apps packages in this environment.
```

Comment:
nit: can we update both this comment and the one at the top of the block to instead reference installing misc js build dependencies?

Reply:
Won't `bundle` not have mini_racer in the ruby path even if the gem isn't removed from disk?

Reply:
FWIW, I manually manipulated the Gemfile and Gemfile.lock on test to remove mini_racer (but didn't explicitly `gem uninistall` it) and verified that a test run succeeded, AND that `bin/dashboard-console` was now using the node external for execjs.